packages/story-editor/src/components/library/panes/shopping/shoppingPane.js

```diff
@@ -236,20 +265,31 @@ function ShoppingPane(props) {
             <ProductSort onChange={onSortBy} sortId={`${orderby}-${order}`} />
           </Row>
         )}
-        {isLoading && (
-          <Loading>
-            <Spinner>
-              <CircularProgress size={24} />
-            </Spinner>
-          </Loading>
+        {!loaded && isLoading && (
+          <LoadingContainer>
+            <LoadingSpinner animationSize={50} circleSize={6} />
+          </LoadingContainer>
         )}
```

Comment:
Now there is no spinner anymore when searching for products or sorting them. Let's ensure that we keep displaying a spinner in these cases.

Reply:
It doesn't have a spinner as it does not need one anymore.

Reply:
Why don't we _need_ a spinner anymore when searching?

The spinner was a useful indicator for the user to know that something is happening.